app/forms/nominate_induction_tutor_form.rb

```diff
@@ -6,7 +6,7 @@ class NominateInductionTutorForm
   attr_accessor :full_name, :email, :token, :school_id, :user_id
 
   validates :full_name, presence: true
-  validates :email, presence: true, format: { with: Devise.email_regexp }
+  validates :email, presence: true, notify_email: true
```

Comment:
How does this work? How does it get wired and told to use `NotifyEmailValidator`?

Reply:
Rails magic of course! It's just based on the name, and that it inherits from `ActiveModel::EachValidator`

Reply:
Your favourite!

Reply:
Aaaaarghhhhhhh, my head has just blown up.